frontend/src/Pages/Team/Card.jsx

```diff
@@ -1,23 +1,28 @@
 import React from 'react'
 import './TeamPage.styles.css'
+import linkedinIcon from '../../Assets/linkedin-icon.png'
 
+const Card = ({ spacingId, name, designation, imageUrl }) => {
+    const offSpaced = [2, 5, 8, 11, 13, 14, 16, 17, 18]
 
-const Card = ({name, designation,imageUrl}) => {
-   return (
-      <div class="col s12 m6 l4 ">
-         <div id="mobcard">
-            <div className="card team_card">
-               <div className="card-image">
-                  <img src={imageUrl} alt={name} style={{"height": '25em'}} />
-                  <span className="card-title">{name}</span>
-               </div>
-               <div className="card-content team_card_content center">
-                  <p className="designation">{designation}</p>
-               </div>
+    return (
+        <div className={[17, 18].includes(spacingId) ? "col s12 m6 l6" : spacingId === 9 ? "col s12 m12 l4" : "col s12 m6 l4"} >
```

Comment:
Oooh, some magic is happening here ✨.
Can U simplify it for us mortals to understand it please @Vishruth-S 🙈😅

Reply:
Well, from the design, you can see that the columns are not evenly spaced. So one workaround is to create separate components for each of them, but that is unnecessary duplication of code. 
Instead, I found that the id or number of each card was being passed, so I just exploited that and added some conditional logic to get the spacing correct for each card 😉